jaimetacos.md

```diff
@@ -1,7 +1,8 @@
 This is an excellent recipe for wasabi tacos.
 
-- Torillas (corn preferred)
+- Tortillas (corn preferred)
 - Wasabi (freshly made)
 - Poblano peppers
 - Chedder cheese
 - Cream cheese
+- Zatar
```

Comment:
Maybe mayonnaise could be added too @jaimemcc? 🤩

Reply:
I think cream cheese *and* mayo would be too much but I have added it as an option instead of the cheese.

Reply:
Great work @jaimemcc ✌